game/src/sandbox/time_warp.rs

```diff
@@ -225,22 +164,41 @@ impl State<App> for JumpToDelay {
                         Some(delay),
                     ));
                 }
-                "Jump to time" => {
-                    return Transition::Replace(JumpToTime::new(ctx, app, self.maybe_mode.take()));
+                action => {
+                    if self.tabs.handle_action(ctx, action, &mut self.panel) {
+                        // if true, tabs has handled the action
+                    } else {
+                        todo!("handle action: {}", action)
```

Comment:
I'm guessing this was an intermediate step of developing the tab controller. Should we change this to an `assert` or something else now that this case should be unreachable?

Reply:
done!